.github/workflows/test.yml

```diff
@@ -42,6 +42,7 @@ jobs:
       - name: Unit Tests
         run: |
           cd install
+          ./create-docker-volumes-test.sh
           ./relay-credentials-test.sh
           cd ..
```

Comment:
```suggestion
        working-directory: install
        run: |
          ./create-docker-volumes-test.sh
          ./relay-credentials-test.sh
```

Reply:
Done in cde39996b19350443764541682b14102b3e84086.